azure/functions/decorators/function_app.py

```diff
@@ -1620,8 +1625,15 @@ def get_functions(self) -> List[Function]:
 
         :return: List of functions in the function app.
         """
-        return [function_builder.build(self.auth_level) for function_builder
-                in self._function_builders]
+        functions = [function_builder.build(self.auth_level)
+                     for function_builder in self._function_builders]
+
+        if all(not function.is_http_function() for function in functions):
+            logging.warning(
+                'Auth level for the function app is not applied to non http '
+                'function app')
```

Comment:
Please add a link to either https://docs.microsoft.com/en-us/azure/azure-functions/functions-bindings-http-webhook-trigger?tabs=in-process%2Cfunctionsv2&pivots=programming-language-python#http-auth or a recommendation on what customer should do - e.g. remove auth_level.